### PR TITLE
KMS-15631 change to the Submit button

### PR DESCRIPTION
**Pull request by MaksymKorniiov**
[IVQ] change to the Submit button

### CODE REVIEW

File: modules/Quiz/resources/css/quiz.css [hunk 1]
```diff
@@ -737,6 +737,13 @@ ol.second-row{
...
+    transition: width 0.3s,height 0.3s,top 0.3s,margin 0.3s ease-in-out;
+}
+.quizDone-cont.small{
+    width: 3vw;
+    height: 4vw;
+    top: 89vh;
+    margin-left: 97%;
```
Comment (eitanavgil) [inline modules/Quiz/resources/css/quiz.css]: @MaksymKorniiov  
Have we reviewed this on fullscreen? How does 97% look in fullscreen mode?

File: modules/Quiz/resources/quiz.js [hunk 2]
```diff
@@ -737,6 +737,9 @@
                     .attr('title', 'click to end quiz now').on('keydown', _this.keyDownHandler).attr('id', 'quiz-done-continue-button').focus();
             // verify focus in IE
             document.getElementById('quiz-done-continue-button').focus();
+            setTimeout(function () {
+                $(_this.embedPlayer.getInterface()).find('.quizDone-cont').first().addClass('small');
+            },3000);
```
Comment (eitanavgil) [inline modules/Quiz/resources/quiz.js]: @MaksymKorniiov  
If we are in playlist - do we need to clear this or is it destroyed naturally by the media change? (I am asking - not commenting)

Comment (OrenMe) [reply modules/Quiz/resources/quiz.js]: @eitanavgil why don't you cache `quizDone-cont`?
you are creating it and then reffereing to it multiple times, but in each place you are using jquery selectors to reach it.
Save ref here:
```
scrubber.parent().prepend('<div class="quizDone-cont"></div>');
```
Then don't put global listeners here:
```
$(document).off( 'click', '.quizDone-cont' )
                 .on('click', '.quizDone-cont', function () {
                     _this.KIVQModule.quizEndScenario();
                 });
```
Unless I'm missing something, but why do we need to put it on window?
and then again in here:
```
$(_this.embedPlayer.getInterface()).find('.quizDone-cont').first().addClass('small');
```
You are searching for the same element again, where you just created it 10 lines above, so better just keep ref to it.

Comment (eitanavgil) [reply modules/Quiz/resources/quiz.js]: @OrenMe  I agree. Although this function is called once. 

@MaksymKorniiov  can you please change that ?

**Pull request closed without merging**